packages/babel-preset-env/package.json

```diff
@@ -90,10 +90,9 @@
     "@babel/plugin-transform-unicode-regex": "workspace:^",
     "@babel/plugin-transform-unicode-sets-regex": "workspace:^",
     "@babel/preset-modules": "0.1.6-no-external-plugins",
-    "@babel/types": "workspace:^",
-    "babel-plugin-polyfill-corejs2": "^0.4.5",
+    "babel-plugin-polyfill-corejs2": "condition:BABEL_8_BREAKING ? : ^0.4.5 (peer:@babel/core)",
```

Comment:
`(peer:@babel/core)` is to correctly propagate the `@babel/core` dep provided by `@babel/preset-env` to `babel-plugin-polyfill-corejs2-BABEL_8_BREAKING-false`.